src/main/java/org/jabref/logic/xmp/DublinCoreExtractor.java

```diff
@@ -78,24 +82,20 @@ private void extractAuthor() {
      * property filled with jan will override this behavior and no data is lost. In the cases, where xmp is written by
      * another service, the assumption is, that the 1st January is not a publication date at all.
      */
-    private void extractYearAndMonth() {
+    private void extractDate() {
         List<String> dates = dcSchema.getUnqualifiedSequenceValueList("date");
         if ((dates != null) && !dates.isEmpty()) {
+
             String date = dates.get(0).trim();
-            Calendar calender = null;
             try {
-                calender = DateConverter.toCalendar(date);
-            } catch (IOException ignored) {
-                // Ignored
-            }
-            if (calender != null) {
-                bibEntry.setField(StandardField.YEAR, String.valueOf(calender.get(Calendar.YEAR)));
-                int monthNumber = calender.get(Calendar.MONTH) + 1;
-                // not the 1st of January
-                if (!((monthNumber == 1) && (calender.get(Calendar.DAY_OF_MONTH) == 1))) {
-                    Month.getMonthByNumber(monthNumber)
-                         .ifPresent(month -> bibEntry.setMonth(month));
-                }
+                Date.parse(date)
+                        .ifPresent(dateValue -> {
+                            dateValue.getDay().ifPresent(day -> bibEntry.setField(StandardField.DAY, Integer.toString(day)));
+                            dateValue.getMonth().ifPresent(bibEntry::setMonth);
+                            dateValue.getYear().ifPresent(year -> bibEntry.setField(StandardField.YEAR, Integer.toString(year)));
+                        });
+            } catch (RuntimeException e) {
+                LOGGER.error("Failed To Parse Date\n {}", ExceptionUtils.getStackTrace(e));
```

Comment:
Is it really necessary to catch a RuntimeEception here? You have already optionals that will evaluate to empty if they cannot parse a date.
A Runtime Exception is an unchecked exception that should never be caught. An exception should always be as precise as possible.
https://www.baeldung.com/java-exceptions 

Second, every logger takes an exception/throwable as second parameter. Stack trace etc are handled automatically